### DISCUSSION TITLE
Dual Monitor Compactibility

### DISCUSSION THREAD

**Discussion by Flwnk120** [mpurses/Sonder#23] [Q&A] [answered]
I have 2 monitors at my disposal and I wanted the same design from my first monitor to be in my second monitor, is there a way to do that?

**Comment by mpurses** [comment, marked as answer]
The easiest way to do that would be to make another copy of the whole Sonder folder (in Documents/Rainmeter/Skins). Name it Sonder2 for example, and hit Refresh All in the main Rainmeter window for it to show up.

**Reply by Flwnk120** [reply]
This should do the job. Thanks a lot.
